<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MsgPopWidget</class>
 <widget class="QWidget" name="MsgPopWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>248</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>439</width>
    <height>248</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>439</width>
    <height>248</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QWidget" name="pop_widget_title" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777214</width>
       <height>35</height>
      </size>
     </property>
     <widget class="QPushButton" name="pop_close_btn">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>3</y>
        <width>24</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="title_lab">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>1</y>
        <width>71</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;
color: rgb(0, 200, 255);</string>
      </property>
      <property name="text">
       <string>提示</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QPushButton" name="pop_commit_btn">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>140</y>
        <width>95</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="ico_lab">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="msg_label">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>261</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="error_code_lab">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>261</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
